srcpkgs/exiv2/template

```diff
@@ -22,8 +23,10 @@ if [ -n "$XBPS_CCACHE" ]; then
 fi
 
 pre_check() {
-	# https://github.com/Exiv2/exiv2/issues/2762
-	rm -f tests/bugfixes/github/test_issue_2427.py
+	# https://github.com/Exiv2/exiv2/issues/3052
+	if [ "$XBPS_TARGET_MACHINE" = i686 ]; then
+		rm -f tests/lens_tests/test_canon_lenses.py
```

Comment:
Don't use `-f`, it hides changes in file structure that should instead trigger removal of obsolete commands.